hexrd/ui/main_window.py

```diff
@@ -563,24 +563,33 @@ def on_action_edit_euler_angle_convention(self):
         self.update_all()
         self.update_config_gui()
 
-    def on_action_edit_apply_polar_mask_triggered(self):
+    def on_action_edit_apply_hand_drawn_mask_triggered(self):
         # Make the dialog
         canvas = self.ui.image_tab_widget.image_canvases[0]
-        self._apply_polar_mask_line_picker = (
+        self._apply_drawn_mask_line_picker = (
             HandDrawnMaskDialog(canvas, self.ui))
-        self._apply_polar_mask_line_picker.start()
-        self._apply_polar_mask_line_picker.finished.connect(
+        self._apply_drawn_mask_line_picker.start()
+        self._apply_drawn_mask_line_picker.finished.connect(
             self.run_apply_polar_mask)
 
-    def run_apply_polar_mask(self, line_data):
+    def run_apply_polar_mask(self, dets, line_data):
```

Comment:
Can we rename this to `run_apply_hand_drawn_mask`?

Reply:
Yes, renamed